x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/add_prepackaged_rules_route.test.ts

```diff
@@ -88,13 +82,11 @@ describe.skip.each([
     mockExceptionsClient = listMock.getExceptionListClient();
 
     context.securitySolution.getConfig.mockImplementation(() =>
-      configMock.withRuleRegistryEnabled(defaultConfig, isRuleRegistryEnabled)
+      configMock.withRuleRegistryEnabled(defaultConfig, true)
     );
 
-    clients.rulesClient.find.mockResolvedValue(getFindResultWithSingleHit(isRuleRegistryEnabled));
-    clients.rulesClient.update.mockResolvedValue(
-      getAlertMock(isRuleRegistryEnabled, getQueryRuleParams())
-    );
+    clients.rulesClient.find.mockResolvedValue(getFindResultWithSingleHit(true));
+    clients.rulesClient.update.mockResolvedValue(getAlertMock(true, getQueryRuleParams()));
```

Comment:
Can `getAlertMock` be refactored to remove the `isRuleRegistryEnabled` param now that we're in `8.0`, or are these situations where it can still be disabled (and tested)?